Source/Scene/Cesium3DTileset.js

```diff
@@ -501,31 +501,43 @@ define([
                 } else {
                     // Tile does not meet SSE.
 
+                    // Save result to avoid doing unnecessary
+                    // isRefinable() multiple times
+                    var tileIsRefinable = t.isRefinable(cullingVolume);
+                    var visibleChildren = [];
+
+                    for (k = 0; k < childrenLength; ++k) {
+                        child = children[k];
+                        if (child.visibility(cullingVolume) !== CullingVolume.MASK_OUTSIDE) {
+                            visibleChildren.push(child);
+                        }
+                    }
```

Comment:
To avoid duplicate operations here and in `isRefinable`, the code could be reformatted like:

```javascript
var visibleChildren = t.getVisibleChildren();
var tileIsRefinable = t.isRefinable(visibleChildren, cullingVolume);
```

Or something with a similar idea.